contractcourt/channel_arbitrator.go

```diff
@@ -612,16 +612,27 @@ func (c *ChannelArbitrator) relaunchResolvers(commitSet *CommitSet,
 	// We'll also fetch the historical state of this channel, as it should
 	// have been marked as closed by now, and supplement it to each resolver
 	// such that we can properly resolve our pending contracts.
-	chanState, err := c.cfg.FetchHistoricalChannel()
-	if err != nil {
+	var chanState *channeldb.OpenChannel
+	chanState, err = c.cfg.FetchHistoricalChannel()
+	switch {
+	// If we don't find this channel, then it may be the case that it
+	// was closed before we started to retain the final state
+	// information for open channels.
+	case err == channeldb.ErrChannelNotFound:
+		log.Warnf("ChannelArbitrator(%v): unable to fetch historical "+
+			"state", c.cfg.ChanPoint)
+		break
```

Comment:
linter says this is redundant

Reply:
Heh yeh, I like to add them personally since IMO it makes things more explicit. Will remove and add a comment to make out linter overlords happy.